docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,66 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode networking which can be used as a simple replacement for Slirp (which is practically dead).
+
+`passt` is a universal tool which implements a translation layer between a Layer-2 network interface and native
+Layer -4 sockets (TCP, UDP, ICMP/ICMPv6 echo) on a host.<br/>
+The main benefits of `passt` are that it doesn't require extra network capabilities as CAP_NET_RAW and CAP_NET_ADMIN
+and allows integration with service meshes (which expect applications to run locally) out of the box.
+
+Moreover, `passt` supports IPv6 out of the box (in contrast to the existing bindings which require configuring IPv6
+manually).
+
+|                                                                | Masquerade                                                                                            | Bridge                                                                   | Passt                                                                                                |
+|----------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| Supports migration                                             | V                                                                                                     | X                                                                        | X<br/>(will be supported in the future)                                                              |
+| Inner and outer IPs are the same                               | X                                                                                                     | V                                                                        | V<br/>(in the future it will be possible to configure the VM IP. Currently the default is the pod IP) |
```

Comment:
Not sure about inner / outer IPs. I think it would be better to be explicit, and refer what they actually are: pod / VM IPs.

Reply:
done.